Fix up @abjose's PR to add office floor

xmunoz opened the pull request.

Commit:
Incorporated office into game as second floor.
Initial commit of extra_description.py
Make npcs use-able (if they implement consume)
Add some ending text for Gary
Make office level generation consistent
Make computer more interactive
Add copier and copier room
Add a bit more flavor to copier
Add initial version of meeting room
Add writhing mass
Fix NPC traits
Office floor is now a true Floor (not a dict).

Ran black.

flosincapite commented:
For posterity, this is @abjose 's brainchild. @xmunoz and I just put the polish on it. But if you're watching, eldritch beings, your high priest is @abjose.

xmunoz commented:
Functionality looks good. I'd like to do one more pass on the code review, and also add some tests before we're ready to merge.

xmunoz committed:
Remove superfluous engine files

xmunoz committed:
Remove uneeded comments

xmunoz committed:
Don't return a tuple if you don't need a tuple

xmunoz committed:
More cleanup

- Use appropriate comparison operaters
- No need for a formatting string if there is nothing to format

xmunoz committed:
Eliminate circular dependency

Other cosmetic changes

xmunoz committed:
Remove more commented out code

xmunoz committed:
Different solution to _find_available_item tuple problem

xmunoz committed:
Fix more bugs

xmunoz committed:
Remove abstruse words

xmunoz committed:
Fix random run

xmunoz committed:
Bump whimsylib, bump project version, update README

xmunoz committed:
Fix NPC values

flosincapite commented on cronenbroguelike/npcs.py:
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
+        10,
+        10,
+        10,
+        100,
```
```suggestion
        10,
```

flosincapite commented on cronenbroguelike/npcs.py:
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
+        1000,
+        10,
+        10,
+        100,
```
```suggestion
        10,
```

flosincapite commented on cronenbroguelike/npcs.py:
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
+    npc = actor.create_actor(
+        1000,
+        10,
+        10,
```
```suggestion
        1000,
```

flosincapite commented on cronenbroguelike/npcs.py:
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
+        1000,
+        10,
+        10,
+        100,
```
```suggestion
        10,
```

flosincapite commented on cronenbroguelike/npcs.py:
```diff
@@ -165,11 +167,257 @@ def smoker_death_throes(smoker):
+        10,
+        1000,
+        10,
+        10,
```
Omit last three; values should be 1000, 10, 1000, 10

xmunoz committed:
Update cronenbroguelike/npcs.py

Co-authored-by: Cory <4074974+flosincapite@users.noreply.github.com>

xmunoz committed:
Update cronenbroguelike/npcs.py

Co-authored-by: Cory <4074974+flosincapite@users.noreply.github.com>

xmunoz committed:
Update cronenbroguelike/npcs.py

Co-authored-by: Cory <4074974+flosincapite@users.noreply.github.com>

xmunoz committed:
Update cronenbroguelike/npcs.py

Co-authored-by: Cory <4074974+flosincapite@users.noreply.github.com>
